{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by an unexpected incident on our underground cable that provides your supply.",
  "CTAs": [],
  "faultNumber": "60499090",
  "multipleFaultNumbers": null,
  "date": "2020-12-21T16:41:58",
  "region": "Ulverston",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 0,
  "estimatedTimeOfRestoration": null,
  "estimatedTimeOfRestorationMajority": "2020-12-21T16:41:58",
  "information": "",
  "actualTimeOfRestoration": "2020-12-21T16:41:58",
  "outageCentrePoint": {
    "lat": 54.172956,
    "lng": -3.147409
  },
  "outageLocations": null,
  "AffectedPostcodes": " LA12 0AP, LA12 0LH",
  "EtrMUnderReview": true,
  "WebTMSFaultType": "U/GM"
}